src/assets/wise5/authoringTool/project-authoring-step/project-authoring-step.component.ts

```diff
@@ -2,18 +2,21 @@ import { Component, EventEmitter, Input, Output } from '@angular/core';
 import { TeacherProjectService } from '../../services/teacherProjectService';
 import { TeacherDataService } from '../../services/teacherDataService';
 import { Router } from '@angular/router';
+import { SelectNodeEvent } from '../domain/select-node-event';
+import { NodeTypeSelected } from '../domain/node-type-selected';
 
 @Component({
   selector: 'project-authoring-step',
   templateUrl: './project-authoring-step.component.html',
   styleUrls: ['./project-authoring-step.component.scss']
 })
 export class ProjectAuthoringStepComponent {
-  @Input() disableSelection: boolean = false;
-  @Input() item: any;
+  checked: boolean = false;
+  @Input() nodeTypeSelected: NodeTypeSelected;
```

Comment:
ProjectAuthoringLesson and ProjectAuthoringStep shouldn't have to know about NodeTypeSelected- they should only need to know if they need to disable selection?
```suggestion
  @Input() disableSelection: boolean = false;
```